drivers/SmartThings/harman-luxury/src/init.lua

```diff
@@ -407,6 +405,13 @@ driver:call_on_schedule(const.HEALTH_CHEACK_INTERVAL, function()
   update_connection(driver)
 end, const.HEALTH_TIMER)
 
+-- create driver values update routine
+
+log.info("create_check_for_updates_manager for Harman Luxury devices")
+driver:call_on_schedule(const.UPDATE_INTERVAL, function()
+  check_for_updates_manager(driver)
+end, const.UPDATE_TIMER)
```

Comment:
It seems like there is the potential for an explosion of memory/network activity if devices are not being responsive and there are multiple devices in the driver. 

Take the case where the first device in the drivers list of devices is non-responsive. Each call to `check_for_updates_manager` will take more than 5 seconds to complete, but in that time it will be called 5 times. This is not good. 

One way around this, is to spawn a cosock task that has an infinite loop to do the updates and make sure that a full iteration of devices is completed before starting another. However, I think it would be preferable to have a single "task" per device to ensure devices are updated at the update rate if possible, and when being non-responsive do not block other devices from updating.